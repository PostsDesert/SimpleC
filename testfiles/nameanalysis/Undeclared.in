int testFunc() {
    a = 5;
}