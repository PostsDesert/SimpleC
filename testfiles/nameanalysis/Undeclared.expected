Error at line 2: Symbol 'a' is not declared
